packages/snap-preact-components/src/components/Organisms/BranchOverride/BranchOverride.tsx

```diff
@@ -309,6 +312,9 @@ export const BranchOverride = (properties: BranchOverrideProps): JSX.Element =>
 
 				<div
 					className="ss__branch-override__top__collapse"
+					ref={(e) => useA11y(e)}
+					role={'button'}
+					aria-label={'stop searchspring preview'}
```

Comment:
Hmmm... Do we want to make this navigable for accessibility?

Reply:
Yeah wasn't necessary for sure. But I was doing all the other components so felt wrong to skip it. I can delete if you would like.

Reply:
Yea, let's remove this for now.